kolibri/plugins/coach/assets/src/views/attempt-log-list/index.vue

```diff
@@ -49,7 +49,7 @@
       header: 'Answer history',
       today: 'Today',
       yesterday: 'Yesterday',
-      daysAgo: ' { daysElapsed } days ago',
+      daysAgo: '{ daysElapsed } days ago',
```

Comment:
Perhaps we could add number formatting to this also?

`'{ daysElapsed, number } days ago'`